fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugDrawer.kt

```diff
@@ -115,11 +105,30 @@ private fun topBarBackButton(onClick: () -> Unit): @Composable () -> Unit = {
 @LightDarkPreview
 private fun DebugDrawerPreview() {
     val navController = rememberNavController()
+    val destinations = remember {
+        List(size = 15) { index ->
+            DebugDrawerDestination(
+                route = "screen_$index",
+                title = R.string.debug_drawer_title,
+                content = {
+                    Text(
+                        text = "Tool $index",
+                        color = FirefoxTheme.colors.textPrimary,
+                        style = FirefoxTheme.typography.headline6,
+                    )
```

Comment:
This dummy destination creation should go away when AC-level, general-purpose features are added to the drawer. When that happens, we'll be able to create another `DebugDrawerRoute` style of enum to simplify all of this.